rfc8826.xml

```diff
@@ -1214,70 +1380,133 @@ the title) for use on https://www.rfc-editor.org/search. -->
             <author initials="C." surname="Jackson">
               <organization/>
             </author>
-            <!-- Date from PDF properties -->
             <date month="May" year="2011"/>
           </front>
-         <refcontent>W2SP, 2011</refcontent>
+          <refcontent>Web 2.0 Security and Privacy (W2SP 2011)</refcontent>
         </reference>
 
         <reference anchor="finer-grained">
           <front>
             <title>Beware of Finer-Grained Origins</title>
-            <author initials="A." surname="Barth">
+            <author initials="C." surname="Jackson">
               <organization/>
             </author>
-            <author initials="C." surname="Jackson">
+            <author initials="A." surname="Barth">
               <organization/>
             </author>
-            <!-- Date from PDF properties -->
             <date month="July" year="2008"/>
           </front>
-         <refcontent>W2SP, 2008</refcontent>
+          <refcontent>Web 2.0 Security and Privacy (W2SP 2008)</refcontent>
         </reference>
 
-        <reference anchor="CORS" target="http://www.w3.org/TR/cors/">
+<!-- [rfced] Is the [CORS] reference still correct?  Should this document
+instead refer to <https://fetch.spec.whatwg.org/>?  Perhaps
+<https://fetch.spec.whatwg.org/#http-cors-protocol>, more specifically? 
+
+Original:
+   [CORS]     van Kesteren, A., "Cross-Origin Resource Sharing", January
+              2014.
+
+When we search for this document, we find this link
+<https://www.w3.org/TR/2009/WD-cors-20090317/>, which gives the following
+warning: 
+
+   This version is outdated!
+   For the latest version, please look at https://www.w3.org/TR/cors/. 
+
+<https://www.w3.org/TR/cors/> redirects to <https://fetch.spec.whatwg.org/>.
+
+On <https://fetch.spec.whatwg.org/>, the reference for [CORS] refers back to
+<https://www.w3.org/TR/cors/>:
+
+[CORS]
+    Anne van Kesteren. Cross-Origin Resource Sharing. 2 June 2020. REC. URL:
+    https://www.w3.org/TR/cors/ 
+
+<https://www.w3.org/TR/2020/SPSD-cors-20200602/> says that new implementations
+should follow the "Fetch API Living Standard".
+
+Please review and let us know if any updates are needed. 
```

Comment:
Yes, update please.

Reply:
@annevk: can you help here?

Reply:
I think this should be as simple as replacing the CORS reference with a reference to Fetch (which includes a definition of the CORS protocol).

Following the format above, something like:

```
[FETCH]

    Anne van Kesteren. Fetch. Living Standard. URL:
    https://fetch.spec.whatwg.org/
```